tests/test_faster_rcnn.py

```diff
@@ -46,8 +46,10 @@ def config():
     return cfg
 
 
-def test_faster_rcnn_train_one_epoch(config, dataset):
+@patch("datasetinsights.io.tracker.factory.TrackerFactory.create")
+def test_faster_rcnn_train_one_epoch(mock_create, config, dataset):
```

Comment:
why should this test be changed rather than a different test be created that actually checks the factory is called to get a tracker?

Reply:
I am only making change to mock factory.create call because of following two reasons: 
1. We are creating estimator object under this method which calls factory.create so do want to make call outside of the estimator constructor.
2. Also, under train_one_epoch, I am doing mltracking logging and I don't want to do logging when I run unit test case so need to mock this call.